packages/schematics/angular/utility/latest-versions.ts

```diff
@@ -6,6 +6,18 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+/** Retrieve the minor version for the provided version string. */
+function getAngularEarliestMinorVersion(version: string): string {
```

Comment:
Wouldn't this cause CI failures again when we change the major/minor version?

Reply:
Left as hardcoded for now.
Once the build system is fully transitioned to Bazel, we can use stamping to set the version automatically at build time.